src/pe/header.rs

```diff
@@ -391,17 +391,96 @@ impl DosHeader {
 ///
 /// * De facto, can be followed by a non-standard ["Rich header"](https://0xrick.github.io/win-internals/pe3/#rich-header).
 /// * According to the standard, is followed by the  [Header::signature] and then the [CoffHeader].
-pub struct DosStub(pub [u8; 0x40]);
-impl Default for DosStub {
+pub struct DosStub<'a> {
+    pub data: &'a [u8],
```

Comment:
I'm going to change this to private and have a method `bytes() -> &'a [u8]` in a separate patch; if we ever want to switch this to something like `Cow<'a, [u8]>` and have an owned version to allow `Header<'a>` will be easier